demo/laneLoader.js

```diff
@@ -528,21 +548,78 @@ function updateSegments(laneSegments, clonedBoxes, prevIsContains, point, index,
   return newIsContains
 }
 
+
+// Adds anomaly annotations
+function addAnnotations(laneGeometries) {
+  let point;
+
+  let aRoot = viewer.scene.annotations;
+  let aAnomalies = new Potree.Annotation({
+    title: "Lane Anomalies",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  aAnomalies.visible = false;
+  aRoot.add(aAnomalies);
+  let aLeft = new Potree.Annotation({
+    title: "Left",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  let aRight = new Potree.Annotation({
+    title: "Right",
+    position: laneGeometries.rightAnomalies[0]
+
+  });
+  aAnomalies.add(aLeft);
+  aAnomalies.add(aRight);
+
+  // Add anomalies layer
+  for (let ii=0, len=laneGeometries.leftAnomalies.length; ii<len; ii++) {
+    point = laneGeometries.leftAnomalies[ii];
+    let aAnomaly = new Potree.Annotation({
+      title: "L " +[ii+1],
+      position: point,
+      cameraPosition: new THREE.Vector3(point.x, point.y, point.z + 20),
+      cameraTarget: point
+    });
+    aLeft.add(aAnomaly);
+  }
+  for (let ii=0, len=laneGeometries.rightAnomalies.length; ii<len; ii++) {
+    point = laneGeometries.rightAnomalies[ii];
```

Comment:
`const point =` I don't want to think about whether this point has any relation to the previous one. (It doesn't)